.storybook/config.js

```diff
@@ -1,6 +1,9 @@
 import { configure } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
 
+/* eslint-disable import/no-dynamic-require, global-require */
 function loadStories() {
-  require('../src/app/components/Article.stories.jsx'); // eslint-disable-line import/no-dynamic-require, global-require
+  require('../src/app/components/Article.stories.jsx');
+  require('../src/app/components/Header/index.stories.jsx');
```

Comment:
The header changes shouldn't be done in the same PR, this can be a small PR that patches the former error, it shouldn't be in a feature delivery.